.ci/pitest.sh

```diff
@@ -150,6 +150,7 @@ pitest-javadoc)
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        beginJavadocTree(root);</span></pre></td></tr>"
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        finishJavadocTree(root);</span></pre></td></tr>"
   "AbstractJavadocCheck.java.html:<td class='covered'><pre><span  class='survived'>        javadocTokens.clear();</span></pre></td></tr>"
+  "JavadocParamOrderCheck.java.html:<td class='covered'><pre><span  class='survived'>      col += startColumnNumber;</span></pre></td></tr>"
```

Comment:
Is this required? I don't see it is being implemented in check. Pitest 2 might failing be failing due this.

Reply:
yes removed already waiting for rest 3 pending checks then i will send update